contributed_definitions/nyaml/NXraman.yaml

```diff
@@ -118,6 +133,7 @@ NXraman(NXoptical_spectroscopy):
       doc: |
         Specify the type of Raman experiment.
       enumeration: [in situ Raman spectroscopy, resonant Raman spectroscopy, non-resonant Raman spectroscopy, Raman imaging, Tip-enhanced Raman spectroscopy (TERS), Surface-enhanced Raman spectroscopy (SERS), Surface plasmon polariton enhanced Raman scattering (SPPERS), Hyper Raman spectroscopy (HRS), Stimulated Raman spectroscopy (SRS), Inverse Raman spectroscopy (IRS), Coherent anti-Stokes Raman spectroscopy (CARS), other]
```

Comment:
consistence, instead of space separated terms with abbreviation why not all lower-case and underscore i.e. in_situ_raman_spectroscopy

Reply:
thanks for this catch: I changed all enumerations to lowercase (except for names (Raman) and acronyms (CCD))

Reply:
us yea, just didnt respond at all to the underscore stuff back then. But, yea - as discussed, that the lowercase it not necessary.